draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -43,75 +43,37 @@ organization="Ping Identity"
-variety of applications. SD-JWT is a selective disclosure mechanism for JWT and is
-similarly intended to be a general-purpose specification.
-
-While JWTs with claims describing natural persons are a common use case, the
-mechanisms defined in this document are also applicable to other use cases.
-
-In an SD-JWT, claims can be hidden, but cryptographically
-protected against undetected modification. When issuing the SD-JWT to
-the Holder, the Issuer includes the cleartext counterparts of all hidden
-claims, the so-called Disclosures, outside the signed part of the SD-JWT.
-
-The Holder decides which claims to disclose to a particular Verifier and includes the respective
-Disclosures in the SD-JWT to that Verifier. The Verifier
-has to verify that all disclosed claim values were part of the
-Issuer-signed JWT. The Verifier will not, however, learn any hidden claim
-values not disclosed in the Disclosures.
-
-This document also defines a format for SD-JWTs with Key Binding (SD-JWT+KB).
-By optionally sending an SD-JWT+KB to a
-Verifier, the Holder can prove to the Verifier that they hold the private key
-associated to the SD-JWT (i.e., using the `cnf` claim [@!RFC7800]). The strength of the binding is conditional upon the trust
-in the protection of the private key of the key pair an SD-JWT is bound to.
-
-SD-JWT can be used with any JSON-based representation of claims.
-
-This specification aims to be easy to implement and to leverage
-established and widely used data formats and cryptographic algorithms
-wherever possible.
+JSON-encoded data structures for exchange between systems are often secured against modification using JSON Web Signatures (JWS) [@!RFC7515].
+A popular application of JWS are JSON Web Tokens (JWT) [@!RFC7519], which are often used to represent a user's identity.
+An ID Token as defined in OpenID Connect [@?OpenID.Core], for example, is a JWT containing the user's claims created by the server for consumption by a client.
+In case the JWS is sent directly from the server to the client, as in OpenID Connect, the server can select which user claims to include in the JWT, minimizing the information shared with the client.
+
+A new model is emerging that decouples the issuance of a JWT from its presentation.
+In this model, a JWT containing many claims is issued to an intermediate party, who holds the JWT (the Holder).
+The Holder, typically an end-user, can then present the JWT to different verifying parties (Verifiers), who each may only require a subset of the claims in the JWT.
+For example, the JWT may contain claims representing both an address and a birthdate.
+The Holder would like to disclose only their address to one Verifier, and only their birthdate to a different Verifier.
+
+This model requires a mechanism enabling selective disclosure of data elements while ensuring that Verifiers can still check the authenticity of the data provided.
```

Comment:
agree that it should be more explicit that selective disclosure becomes an important privacy feature because the same JWT can be used for different purposes.

Reply:
something like this?
```suggestion
Privacy principles of minimal disclosure in conjunction with this model demand a mechanism enabling selective disclosure of data elements while ensuring that Verifiers can still check the authenticity of the data provided.
```